ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -62,6 +62,10 @@ def Cholesky(*, input, name="Cholesky"):
     return ivy.astype(ivy.cholesky(input), input.dtype)
 
 
+def Ceil(*, x, name="ceil"):
```

Comment:
Doesn't really make a difference but the default value for name is None in the original tensorflow function.

Reply:
@AnnaTz Thanks, I just read the documentation and I have fixed it.

Reply:
Hi, you should also remove the * argument. Arguments of frontend functions should be exactly the same as in the original function.